### DISCUSSION TITLE
Fundamentals Kick-off!

### DISCUSSION THREAD

**Discussion by fboucher** [microsoft/Power-Platform-For-Beginners#1] [Fundamentals of Power Platform]
Do you have a section of the [template](https://github.com/microsoft/Power-Platform-For-Beginners/tree/main/templates) you would like to start working on?
Let your teammates know!

Then you can **Fork** the repo and start creating content!

**Comment by PowerRobin** [comment]
Hey Alberto, this is Robin!
What's your github username?

**Comment by fboucher** [comment]
Hi @PowerRobin 
Any thoughts on how you would teach using a project-based lesson? Like building something in a step-by-step approach. What would that project be and where would you like to start?

**Reply by PowerRobin** [reply]
I think project management is a great starting point, because people from nearly every department & industry can relate to.

I usually start teaching from the data model.
Next stop would be the main app (Model Driven to manage the projects).
From there you could dive into a field app (Canvas App) that allows you to take pictures.
Power Automate for approvals or recurring tasks.
Power BI for a monthly status report.
Power Pages to get the clients into the mix.
PVA as another way to get insights about the data.

Could be a great way for the fundamentals chapter. Instead of giving an overview about every product, show in one solution, when to use what. But we should probably discuss this with the whole group, to use this as a recurrent theme.

**Reply by fboucher** [reply]
IF you have an idea clear enough in your head feel free to share it in details in the General Discussion.

**Comment by faludigabor** [comment]
Hi , 

We also brainstormed a bit, what do you think of this? 

•	How to register a tenant? 
o	Registering a developer tenant
o	Developer environment in existing environment
•	Environment strategy overview
o	Dev / TEST / Prod
o	ALM - concept
•	What are Power Platform components	
•	How to select the right component for the project
•	What the typical use cases of Power Platform
•	How would you get started from existing material
•	Typical Users of Power Platform
o	Citizen Developer
o	Architect
o	.NET Developer
•	Where do I find reusable stuff?
•	Licensing – should we talk about it?

**Reply by fboucher** [reply]
@faludigabor Looks good. Did you join the team Teams?

**Comment by fboucher** [comment]
I'll close this discussion now that most happens on Teams and keep only the more general ones open for now. :)
